config/fiab.json

```diff
@@ -9,7 +9,7 @@
   "googleClientId": "500025638838-s2v23ar3spugtd5t2v1vgfa2sp7ppg0d.apps.googleusercontent.com",
   "jobManagerUrlRoot": "https://job-manager.dsde-dev.broadinstitute.org/jobs",
   "leoUrlRoot": "https://leonardo-fiab.dsde-dev.broadinstitute.org:30443",
-  "marthaUrlRoot": "https://martha-fiab.dsde-dev.broadinstitute.org:32443/fileSummaryV1",
+  "marthaUrlRoot": "https://martha-fiab.dsde-dev.broadinstitute.org:32443/dockerized-martha/us-central1/martha_v2",
```

Comment:
The previous URL for Martha was giving 404s, so changed it to this to make it work for a fiab